.circleci/config.yml

```diff
@@ -122,6 +130,66 @@ commands:
             else
               echo 'user-halt' > "$CIRCLE_INTERNAL_SCRATCH/HALT_TASK"
             fi
+  checkout-shallow:
+    steps:
+      - run:
+          name: Checkout code (shallow)
+          command: |
```

Comment:
Maybe add a comment that you copy-pasted the CircleCI code and slightly modified it :)